scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -328,11 +328,13 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
     accept[RightBrace]
     ret
   }
-  @inline final def inBracesOrError[T](body: => T, alt: T): T =
+  @inline final def inBracesOrError[T](body: => T, alt: T): T = {
+    if (token.is[LF]) { accept[LF] }
```

Comment:
What happens if there are multiple `LF`?

Reply:
it will generate `LFLF` which is invalid in this scenario